site/docs/v1/tech/apis/applications.adoc

```diff
@@ -162,6 +163,125 @@ include::docs/v1/tech/apis/_standard-put-response-codes.adoc[]
 
 include::docs/v1/tech/apis/_application-response-body.adoc[]
 
+== Search for Applications
+
+[NOTE.since]
+====
+This API has been available since 1.44.0
+====
+
+This API is used to search for Applications and may be called using the `GET` or `POST` HTTP methods. Examples of each are provided below. The `POST` method is provided to allow for a richer request object without worrying about exceeding the maximum length of a URL. Calling this API with either the `GET` or `POST` HTTP method will provide the same search results given the same query parameters.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Applications using the given search and sort criteria
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/application/search``?name=\{name\}&active=\{active\}&tenantId=\{tenantId\}&orderBy={orderBy}&startRow={startRow}``#
+--
+
+==== Request Parameters
+
+[field]#name# [type]#[String]# [optional]#Optional#::
+The string to search for in the Application name. This can contain wildcards using the asterisk or percent characters (* or %). This search parameter is case-insensitive.
+
+[field]#active# [type]#[Boolean]# [optional]#Optional#::
```

Comment:
What is the default here?

Reply:
Looks like it is to retrieve all applications. Maybe put a note to see the body for the default, like we do for this field: `application.samlv2Configuration.logout.keyId`.

Reply:
I feel that omitting a search field implies that we're not going to filter on that field, which is different than our other "defaults" where we actually force in a value. We could add a `Defaults to [see description]` blurb, but I feel that it adds noise. 

Anyway, just my $.02. Happy to something here if you'd like.

Reply:
I agree - this is a case where an optional does not have a default value. W/out any search criteria, you end up with just a paginated list of applications using a documented order by.